.circleci/config.yml

```diff
@@ -604,20 +604,21 @@ jobs:
           key: claimable-token-deps-{{ checksum "solana-programs/claimable-tokens/program/Cargo.toml" }}
       - restore_cache:
           keys:
-            - anchor-node-deps-{{ checksum "solana-programs/anchor/audius-data/yarn.lock" }}
+            - anchor-node-deps-{{ checksum "solana-programs/anchor/audius-data/package-lock.json }}
             - anchor-node-deps-
       - run:
           name: anchor tests
           command: |
+            export PROTOCOL_DIR="/home/circleci/project"
             export PATH="/home/circleci/.local/share/solana/install/active_release/bin:/usr/local/bin:$PATH"
             solana-keygen new --no-bip39-passphrase
-            cd solana-programs/anchor/audius-data
-            yarn
+            cd "$PROTOCOL_DIR/solana-programs/anchor/audius-data"
```

Comment:
this should almost certainly still be yarn https://book.anchor-lang.com/chapter_2/installation.html